bodhi/server/tasks/handle_side_and_related_tags.py

```diff
@@ -20,67 +20,49 @@
 import logging
 import typing
 
-import koji  # noqa: 401
+from bodhi.server import buildsys
 
-from bodhi.server import buildsys, util
-from bodhi.server.models import Update
 
 log = logging.getLogger(__name__)
 
 
-def main(aliases: typing.List[str], from_tag: str):
+def main(builds: typing.List[str],
+         pending_signing_tag: str,
+         from_tag: str,
+         pending_testing_tag: typing.Optional[str]):
```

Comment:
If it's optional shouldn't it have a default value?

Reply:
Optional means it can be either the type specified (str here) or None. So I think having the default value is fine.